code/datums/gamemode/factions/bloodcult/bloodcult_flavourlines.dm

```diff
@@ -0,0 +1,356 @@
+		=> Your colleagues are in the cult
+		=> Cult is near the end (Act III or higher)
+		=> Cult has a few/a lot of alive members
+*/
+
+var/list/all_depts_list = list(
+	COMMAND_POSITIONS,
+	ENGINEERING_POSITIONS,
+	MEDICAL_POSITIONS,
+	SCIENCE_POSITIONS,
+	CIVILIAN_POSITIONS,
+	CARGO_POSITIONS,
+	SECURITY_POSITIONS,
+)
+
+/datum/faction/bloodcult/proc/send_flavour_text_refuse(var/mob/victim, var/mob/converter)
+	// -- Static context
+	// Default lines
+	var/list/valid_lines = list(
+		"Not worthy of the gift." = 1,
+		"A shame. Maybe you will see our light one day." = 1,
+		"Unconquered in spirit... and destroyed in the flesh." = 1,
+		"So much courage... for nothing." = 1,
+	)
+	// The departement
+	var/victim_job = victim?.mind.assigned_role
+	var/converter_job = converter?.mind.assigned_role
+	for (var/list/L in failure_lines_by_dept)
+		if (victim_job in L)
+			valid_lines += failure_lines_by_dept[L]
+	// The specific job
+	valid_lines += failure_lines_by_specific_job[victim_job]
+	// The roles he may add
+	if (victim.mind)
+		for (var/role in victim.mind.antag_roles)
+			valid_lines += failure_lines_by_specific_role[role]
+	// The race
+	if (ishuman(victim))
+		valid_lines += failure_lines_by_specific_race[victim:species.type]
+
+	// -- Dynamic context
+	// Cultist count
+	var/cultists = 0
+	for (var/datum/role/R in members)
+		if (R.antag && R.antag.current && !R.antag.current.stat) // If he's alive
+			cultists++
+
+	// Not a lot of cultists...
+	if (cultists < 3)
+		valid_lines += failure_lines_few_cultists
+
+	// Or a lot of them !
+	else if (cultists > 10)
+		valid_lines += failure_lines_numerous_cultists
+
+	// Converter and victim are of the same dept
+	for (var/list/dept in all_depts_list)
+		if (victim_job in dept && converter_job in dept)
+			valid_lines += failure_lines_same_dept
+	// Act
+	if (veil_thickness >= CULT_ACT_III)
+		valid_lines += failure_lines_thin_veil
+
+	if(victim.mind && victim.mind.assigned_role == "Chaplain")
+		var/list/cult_blood_chaplain = list("cult", "narsie", "nar'sie", "narnar", "nar-sie")
+		var/list/cult_clock_chaplain = list("ratvar", "clockwork", "ratvarism")
+		if (religion_name in cult_blood_chaplain)
+			to_chat(victim, "<span class='game say'><span class='danger'>Nar-Sie</span> murmurs, <span class='sinister'>Rejoice, I will give you the ending you desired.</span></span>")
+		else if (religion_name in cult_clock_chaplain)
+			to_chat(victim, "<span class='game say'><span class='danger'>Nar-Sie</span> murmurs, <span class='sinister'>I will take your body, but when your soul returns to Ratvar, tell him that [pick(\
+				"... he SUCKS!",\
+				"there isn't enough place for the two of us on this plane!",\
+				"he'll never be anything but a lame copycat.")]</span></span>")
+
+	var/chosen_line = pickweight(valid_lines)
+	var/nar_sie_tone = pick("", "mocking", "cold", "uncaring", "amused")
+	to_chat(victim, "You can hear Nar'Sie's [nar_sie_tone] voice... <span class='warning'>[chosen_line]</span>")
```

Comment:
Nar-Sie, not Nar'Sie, you even spelled it differently on the next line! That being said, there're a few places in the code with the apostrophe instead of the hyphen, so it could do with being made consistent later.